source/material_model/interface.cc

```diff
@@ -255,6 +255,49 @@ namespace aspect
     {}
 
 
+    template <int dim>
+    MaterialModelInputs<dim>::MaterialModelInputs(const FEValuesBase<dim,dim> &fe,
+                                                  const Introspection<dim> &introspection,
+                                                  const LinearAlgebra::BlockVector &solution_vector)
+      :
+      position(fe.get_quadrature_points()),
+      temperature(fe.n_quadrature_points, numbers::signaling_nan<double>()),
+      pressure(fe.n_quadrature_points, numbers::signaling_nan<double>()),
+      pressure_gradient(fe.n_quadrature_points, numbers::signaling_nan<Tensor<1,dim> >()),
+      velocity(fe.n_quadrature_points, numbers::signaling_nan<Tensor<1,dim> >()),
+      composition(fe.n_quadrature_points, std::vector<double>(introspection.n_compositional_fields, numbers::signaling_nan<double>())),
+      strain_rate(fe.n_quadrature_points, numbers::signaling_nan<SymmetricTensor<2,dim> >()),
+      cell (NULL)
```

Comment:
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc:270:7: error: class ‘aspect::MaterialModel::MaterialModelInputs<dim>’ does not have any field named ‘fe’
       fe.get_cell()
       ^
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc:270:9: error: expected ‘(’ before ‘.’ token
       fe.get_cell()
         ^
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc:270:9: error: expected ‘{’ before ‘.’ token
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc: At global scope:
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc:270:9: error: expected unqualified-id before ‘.’ token
/home/paulbremner/install/ASPECT/aspect/devel/source/material_model/interface.cc:760:1: error: expected ‘}’ at end of input
 }
 ^